src/Mpociot/Blacksmith/Models/User.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Mpociot\Blacksmith\Models;
+
+class User extends ForgeModel
+{
+    /**
+     * @return array
+     */
+    public function toArray()
```

Comment:
Why does this function exist in the `User` model. Its already in the extended `ForgeModel` and does the exact same. Its OK to have an empty model. It looks to me like you have copied the `Server` model, which also needs it removing from.